packages/components/package.json

```diff
@@ -46,11 +46,19 @@
     "react-router-dom": "^5.2.0",
     "time-input-polyfill": "^1.0.7",
     "ts-xor": "^1.0.8",
-    "uuid": "^8.3.2"
+    "uuid": "^8.3.2",
```

Comment:
I also no longer think these are needed. Going to verify and remove if no longer needed